frontend/src/component/admin/auth/SamlAuth/SamlAuth.tsx

```diff
@@ -246,6 +253,53 @@ export const SamlAuth = () => {
                         />
                     </Grid>
                 </Grid>
+                <Grid hidden={ssoSyncHidden} container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Enable Group Syncing</strong>
+                        <p>
+                            Enables automatically syncing of users from the SAML provider when a user logs in
+                        </p>
+                    </Grid>
+                    <Grid item md={6} style={{ padding: '20px' }}>
+                        <FormControlLabel
+                            control={
+                                <Switch
+                                    onChange={updateGroupSyncing}
+                                    value={data.enableGroupSyncing}
+                                    disabled={!data.enabled}
+                                    name="enableGroupSyncing"
+                                    checked={data.enableGroupSyncing}
+                                />
+                            }
+                            label={
+                                data.enableGroupSyncing
+                                    ? 'Enabled'
+                                    : 'Disabled'
+                            }
+                        />
+                    </Grid>
+                </Grid>
+                <Grid hidden={ssoSyncHidden} container spacing={3} mb={2}>
+                    <Grid item md={5}>
+                        <strong>Group Field JSON Path</strong>
+                        <p>
+                            Specifies the path in the SAML token response from which to read the groups the user belongs to
+                        </p>
+                    </Grid>
+                    <Grid item md={6}>
+                        <TextField
+                            onChange={updateField}
+                            label="Group JSON Path"
+                            name="groupJsonPath"
+                            value={data.groupJsonPath}
+                            disabled={!data.enableGroupSyncing}
+                            style={{ width: '400px' }}
+                            variant="outlined"
+                            size="small"
+                            required
+                        />
+                    </Grid>
+                </Grid>
```

Comment:
Comments on `OidcAuth.tsx` also apply here. This could also be a good candidate for a separate component so we don't repeat ourselves.

Reply:
So I've extracted only the new properties and put them behind a conditionally render flag. There are a few common properties between the SSO providers that already exist but right now I'd rather have the new stuff isolated rather than extract working things for the sake of refactoring. I think we should probably beat the SSO provider frontend with a big refactor stick when we do a v5 of Unleash and leave working things alone for now. Happy to be told this is crazy talk though